pwndbg/gdblib/__init__.py

```diff
@@ -4,3 +4,18 @@
 from pwndbg.gdblib.arch import arch
 
 __all__ = ["ctypes", "memory", "typeinfo"]
+
+
+# TODO: should the imports above be moved here?
```

Comment:
Not sure what the best way to deal with this is. The above imports are there so that we can rename `pwndbg.gdblib.arch` to `arch_mod` and export the `pwndbg.gdblib.arch` object as `arch`. Maybe the right way to do it is to make them globals, and then assign to those globals inside `load_gdblib`?